CMakeLists.txt

```diff
@@ -79,9 +79,6 @@ else()
 	message(FATAL_ERROR "SENTRY_BACKEND must be one of 'crashpad', 'inproc', 'breakpad' or 'none'")
 endif()
 
-if(SENTRY_BACKEND_CRASHPAD AND NOT (APPLE OR WIN32))
```

Comment:
I would rather flag this with `AND ANDROID`, since we don’t test/support that backend on android yet.

Reply:
Good callout.